lib/common/utils.cpp

```diff
@@ -50,8 +52,9 @@ size_t Wasp::LinearizeCoords(
 size_t Wasp::LinearizeCoords(
 	const size_t *coords, const size_t *dims, int n
 ) {
-	size_t* min = new size_t[n];
-	size_t* max = new size_t[n];
+	VAssert(n <= MAXCOORDS);
+	size_t* min = new size_t[MAXCOORDS];
```

Comment:
@sgpearse I think you misunderstood my comment. These quantities should not be allocated off the heap because it's expensive and LinearizeCoords is often called inside large loops. They should be statically allocated from the stack:

size_t min[MAXCOORDS]
size_t max[MAXCOORDS]

Reply:
Got it.  Fixed.